components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -2965,6 +2966,64 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The Tenant Theme resource APIs
+######################################################
+  /import/tenant-theme:
+
```

Comment:
tenant theme is the resource here. Hence shall we use tenant-theme/upload or import?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/8595/commits/fb8159849c21f633958ff21963a55d40da2ca155